lib/otdoa_al/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+zephyr_library()
+
+# lfs.h is not in the include path, so add it explicitly here
+zephyr_library_include_directories(${ZEPHYR_BASE}/../modules/fs/littlefs)
```

Comment:
this does not make sense, in `zephyr/modules/littlefs/CMakeLists.txt` is:
```
if(CONFIG_FILE_SYSTEM_LITTLEFS)
  add_library(LITTLEFS INTERFACE)

  target_include_directories(LITTLEFS INTERFACE
    ${ZEPHYR_LITTLEFS_MODULE_DIR}
    ${CMAKE_CURRENT_SOURCE_DIR}
  )
  target_link_libraries(LITTLEFS INTERFACE zephyr_interface)
```
so have you included the reference properly?